pkg/kev/project.go

```diff
@@ -151,3 +151,33 @@ func WithUI(ui kmd.UI) Options {
 		project.UI = ui
 	}
 }
+
+func WithK8sNamespace(c string) Options {
```

Comment:
A one liner describing what func is for would be great. Here and in added functions below.